wp-content/themes/pub/wporg-learn-2020/functions.php

```diff
@@ -853,3 +856,32 @@ function wporg_learn_fix_code_entity_encoding( $content ) {
 	return str_replace( '&amp;gt;', '&gt;', $content );
 }
 add_filter( 'syntaxhighlighter_htmlresult', 'wporg_learn_fix_code_entity_encoding', 20 );
+
+/**
+ * Register theme sidebars.
+ */
+function wporg_learn_register_sidebars() {
+	// Register lesson plans sidebar.
+	register_sidebar(
+		array(
+			'name'          => __( 'Lesson Plans', 'wporg-learn' ),
+			'id'            => 'wporg-learn-lesson-plans',
+			'before_widget' => '<div id="%1$s" class="block-widgets %2$s">',
+			'after_widget'  => '</div>',
+			'before_title'  => '',
+			'after_title'   => '',
```

Comment:
We'll probably want some kind of heading markup around dynamic widget titles. Not sure right now what level would be best... maybe `h4`?

Reply:
This has been done.